ykman/openpgp.py

```diff
@@ -375,6 +387,20 @@ def get_remaining_pin_tries(self):
         data = self._get_data(DO.PW_STATUS)
         return PinRetries(*data[4:7])
 
+    def get_signature_pin_policy(self):
+        data = self._get_data(DO.PW_STATUS)
+
+        if bool(data[0]):
+            return POLICY.ONCE
+        else:
+            return POLICY.ALWAYS
```

Comment:
From what I recall, the standard defines this byte as explicitly having the valid values `0x00` and `0x01`, in which case you can just `return POLICY(data[0])`.